app/models/events/review.rb

```diff
@@ -46,5 +46,15 @@ class Review < ApplicationRecord
 
     validates :state, inclusion: { in: states.keys }
     validates :github_id, :opened_at, presence: true
+
+    after_create :build_review_turnaround
```

Comment:
what happens with existing records? are we going to backfill or ignore them?

Reply:
I could build a rake for that. WDYT?

Reply:
Sounds good to me